rubocop.yml

```diff
@@ -0,0 +1,24 @@
+AllCops:
```

Comment:
This file is the `rubocop.yml` file _without_ the `.` at the beginning, so Rubocop isn't actually reading it.  The `.rubocop.yml` file is the real one, and it looks like it already has `Style/ClassAndModuleChildren` disabled in it.